core/vm/contracts.go

```diff
@@ -698,7 +698,7 @@ func (c *transfer) Run(input []byte, caller common.Address, evm *EVM) ([]byte, e
 	//   to:    32 bytes representing the address of the recipient
 	//   value: 32 bytes, a 256 bit integer representing the amount of Celo Gold to transfer
 	// 3 arguments x 32 bytes each = 96 bytes total input
-	if len(input) < 96 {
+	if (evm.chainRules.IsGFork && len(input) != 96) || len(input) <= 96 {
```

Comment:
Hey @karlb was it intended to make this `<=` instead of `<`? This is newly breaking a tracing test I wrote in a draft branch and I think this should be `<` but maybe I'm missing something

Reply:
Yes, you are right! I messed this up when introducing the IsGFork check.